CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing guidelines
+
+Welcome! We're glad to see you here. This short document should give you some hints to get started with contributing to TermPair.
+
+## Getting started
+
+First, fork the repo and clone it to your computer, then read the section you're interested in. 👍
+
+### Contributing Python code
+
+To modify Python code, setup a virtual environment:
+
+```bash
+python -m venv venv
+```
+
+Then activate it with:
+
+```bash
+. venv/bin/activate
+```
+
+You can then install development dependencies using:
+
+```bash
+pip install -e ".[dev]"
+```
+
+### Contributing frontend code
+
+To modify frontend code, get [yarn](https://yarnpkg.com/en/) and run:
+
+```bash
+make build_frontend
+```
+
+You'll need to reload the TermPair server. Stop it with `Ctrl+C` if it's running, then restart it:
```

Comment:
```
yarn install
```
will have to be run once from the frontend_src directory before building the code.

Changes that don't require an actively connected terminal can be tested much more easily with hot reloading by running

```
yarn start
```
from the frontend_src directory.

Reply:
Okay! I see there's a `watch_frontend` Make target to run `yarn start` in `frontend_src`. Should we point to that instead?

Reply:
Yup good call.

Reply:
OK. I'll add an `install_frontend` target too for a more consistent interface. :-)